lemur/static/app/angular/authorities/services.js

```diff
@@ -107,13 +107,7 @@ angular.module('lemur')
     AuthorityService.findActiveAuthorityByName = function (filterValue) {
       return AuthorityApi.getList({'filter[name]': filterValue})
         .then(function (authorities) {
-          var activeAuthorities = [];
-          _.each(authorities, function (authority) {
-              if (authority.active) {
-                activeAuthorities.push(authority);
-              }
-          });
-          return activeAuthorities;
+          return authorities.filter(function(authority) { return authority.active; });
```

Comment:
Internet Explorer 8 and older don't support Array.filter, is that OK?

Reply:
That's fine by me. We don't make any guarantees around browser support.